app/views/home/_signin_publishers.html.haml

```diff
@@ -2,8 +2,8 @@
 
 - if publisher_signed_in?
   %div.manage-vacancies
-    %p= "#{t('pages.home.signed_in.description')} #{current_organisation.name}"
-    = link_to t('pages.home.signed_in.manage_link'), organisation_path, class: 'govuk-button govuk-button--secondary manage-vacancies'
+    %p= "#{t('home.index.signed_in.description')} #{current_organisation.name}"
```

Comment:
Can we use the short lookup `t('.description')` and put the translation under
```
home:
  signin_publishers:
    description: ....
```

Reply:
👍  I was sure these were used somewhere else and that's why they had absolute paths but they're actually not!